packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js

```diff
@@ -78,6 +78,7 @@ export default class RegionsListController {
 
   onMacroChange(macro, regions) {
     if (regions.length === 1) {
+      [this.region] = regions;
       this.onRegionChange(regions[0]);
```

Comment:
In that way you can refer to your previous assignation:

```suggestion
      this.onRegionChange(this.region);
```

Reply:
This is done. Thanks :)